lib/v2/tju/cic/index.js

```diff
@@ -85,16 +81,15 @@ module.exports = async (ctx) => {
         const items = await Promise.all(
             list.map((item) => {
                 switch (item.type) {
-                    case 'wechat-mp':
-                        return finishArticleItem(ctx, item);
                     case 'tju-cic':
                     case 'in-site':
                         return ctx.cache.tryGet(item.link, async () => {
                             let detailResponse = null;
                             try {
-                                detailResponse = await got(item.link, { https: { rejectUnauthorized: false } });
+                                detailResponse = await got(item.link);
                                 const content = cheerio.load(detailResponse.data);
-                                item.description = content('.con_news_body > div').html();
+                                item.pubDate = timezone(parseDate(content('.news_info > span').first().text(), 'YYYY年MM月DD日 HH:mm'), +8);
+                                item.description = content('.v_news_content').html();
```

Comment:
`$('.v_news_content')` in `/tju/cic/notification` does not include the attachments.
I think users will be happier if you can include the attachments in each feed.

e.g., http://cic.tju.edu.cn/info/1041/3876.htm and http://cic.tju.edu.cn/info/1041/3814.htm